platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ExportMigrationConfigurationAdminEntry.java

```diff
@@ -55,7 +58,19 @@ public boolean store() {
     if (!stored) {
       LOGGER.debug(
           "Exporting configuration [{}] to [{}]...", configuration.getPid(), entry.getPath());
-      this.stored = entry.store((r, out) -> persister.write(out, configuration.getProperties()));
+      final Dictionary<String, Object> props = configuration.getProperties();
+
+      // just make sure the pid and factory pid are correct as sometimes they might get out of
+      // sync by an invalid update to the properties and we do rely on those at import time
+      final String factoryPid = configuration.getFactoryPid();
+
+      if (factoryPid != null) {
```

Comment:
❓ Should we log something if we have to update the factoryPid?

Reply:
not really. We end up not caring about these in the dictionary to start with. This is in sync with how felix's config installer works already where it actually removes these properties from the dictionary before saving back to etc or before re-importing from etc.